ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -660,7 +660,27 @@ final class Metrics(val registry: MetricRegistry) {
           def push(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "push")
           def slice(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "slice")
           def prune(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "prune")
+
+          val totalTransactionTreesRetrieved: Counter =
+            registry.counter(Prefix :+ "transaction_trees_retrieved_total")
+          val transactionTreeEventsResolvedFromBuffer: Counter =
+            registry.counter(Prefix :+ "transaction_trees_from_buffer")
+
+          val totalFlatTransactionsRetrieved: Counter =
+            registry.counter(Prefix :+ "flat_transactions_retrieved_total")
+          val flatTransactionEventsResolvedFromBuffer: Counter =
+            registry.counter(Prefix :+ "flat_transactions_from_buffer")
         }
+
+        val getTransactionTreesSource: Timer =
+          registry.timer(Prefix :+ "get_transaction_trees_source")
```

Comment:
```suggestion
          registry.timer(Prefix :+ "get_transaction_trees")
```
In the context of `streams_buffer` there is no ambiguity what it means, same for the next one

Reply:
You're right. Done